src/server/pachyderm_test.go

```diff
@@ -7708,7 +7706,13 @@ func TestListDatumDuringJob(t *testing.T) {
 
 	commit1, err := c.StartCommit(dataRepo, "master")
 	require.NoError(t, err)
-	require.NoError(t, c.PutFile(commit1, "file", strings.NewReader("foo"), client.WithAppendPutFile()))
+
+	fileCount := 10
+	for i := 0; i < fileCount; i++ {
+		fileName := "file" + strconv.Itoa(i)
+		require.NoError(t, c.PutFile(commit1, fileName, strings.NewReader("foo"), client.WithAppendPutFile()))
```

Comment:
Something that is worth noting is that we want to over time remove the `WithAppendPutFile` functional option from calls when it is not needed. What it does is turn the put file into an append operation rather than an overwrite. If the overwrite semantics make sense for the use case then that functional option is unnecessary. The reason why it is in a lot of the test cases now is because we recently changed the semantics of `PutFile` from default append to overwrite, so to avoid figuring out which tests need the append vs not, we defaulted to just including the append option.

Reply:
Makes sense. I'll remove this option here. Thanks!